exonum/src/runtime/dispatcher/mod.rs

```diff
@@ -220,6 +220,20 @@ impl Dispatcher {
             .map_err(From::from)
     }
 
+    /// Initiates stopping of an existing service instance in the blockchain. The stopping
+    /// service is active (i.e., does process transactions or `after_transactions` hook)
```

Comment:
Nit: I'd rephrase as "(i.e., processes transactions and the `after_transactions` hook)".